### PR TITLE
conv2d_transpose + activation(s) fuse pass

### PR DESCRIPTION
**Pull request by Silv3S**
### PR types
Performance optimization

### PR changes
OPs

### Describe
- fuse conv2d_transpose with all supported activations,
- add unit test

### CODE REVIEW

**Comment by Silv3S** [comment]
@zuzg, @piotrekobi please review

**Review by zuzg — APPROVED** [review]
LGTM

File: paddle/fluid/operators/mkldnn/conv_transpose_mkldnn_op.cc [hunk 1]
```diff
@@ -205,27 +205,7 @@ class ConvTransposeMKLDNNHandlerT
...
-          scale, dnnl::algorithm::eltwise_relu, fuse_alpha, fuse_beta);
-    } else if (fuse_activation == "relu6") {
-      constexpr float scale = 1.0f;
-      post_operations.append_eltwise(
-          scale, dnnl::algorithm::eltwise_bounded_relu, fuse_alpha, fuse_beta);
-    } else if (fuse_activation == "swish") {
-      constexpr float scale = 1.0f;
-      post_operations.append_eltwise(
-          scale, dnnl::algorithm::eltwise_swish, fuse_alpha, fuse_beta);
-    }
```
Comment (piotrekobi) [inline paddle/fluid/operators/mkldnn/conv_transpose_mkldnn_op.cc:228]: Wow, that was some ugly code :D. LGTM!

**Review by piotrekobi — APPROVED** [review]

**Comment by Silv3S** [comment]
Internal validation revealed poor performance of fused conv2d_transpose operator. I close the PR until the bug is fixed

**Review by tsocha — APPROVED** [review]

**Pull request closed without merging**